covalent/_workflow/lattice.py

```diff
@@ -343,7 +343,7 @@ def dispatch_sync(self, *args, **kwargs) -> "Result":
 
         return get_result(self.dispatch(**kwargs), self.metadata["results_dir"], wait=True)
 
-    def _server_dispatch(self, result_object) -> str:
+    def _server_dispatch(self, result_object:Result) -> str:
```

Comment:
There should be a space between the parameter and the parameter type. Can you run the tests and ensure that there are no circular import issues? If there are, you might have to use the string representation of result as in line 324.

Reply:
Fixed the issues with circular import by using string representation of result, as you had suggested. I also ran the tests to ensure there were no problems regarding the same.